osu.Game/Screens/Menu/KiaiMenuFountains.cs

```diff
@@ -40,7 +48,7 @@ private void load(AudioManager audio)
                 },
             };
 
-            sample = audio.Samples.Get(@"Gameplay/fountain-shoot");
+            sample = skin.GetSample(new SampleInfo(@"Gameplay/fountain-shoot"));
```

Comment:
Hmm, this either needs to use `SkinnableSound` or handle the `ISkinSource.SouceChanged` flow..

Preferably the former since everything is setup correctly for samples in that class (note that it will need to be added to the draw hierarchy to work correctly).